src/AppBundle/Repository/SwindlerRepository.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace AppBundle\Repository;
+
+class SwindlerRepository extends \Doctrine\ORM\EntityRepository
+{
+    public function findSwindlerIsBanned($ip)
```

Comment:
bad method signature. 
First of all you should add docblock or return type.
If method return boolean method should start with ```is``` prefix, e.g. isIpBanish
If method return Entity|null method should start with ```find``` without ```is``` in method name